pkg/backup/s3/s3.go

```diff
@@ -139,7 +140,7 @@ func (s *S3) CopyPrefix(from string) error {
 	for _, key := range keys {
 		req := &s3.CopyObjectInput{
 			Bucket:     aws.String(s.bucket),
-			Key:        aws.String(path.Join(v1, s.prefix, key)),
+			Key:        aws.String(path.Join(s.prefix, key)),
 			CopySource: aws.String(path.Join(s.bucket, v1, from, key)),
```

Comment:
Also need to change `CopySource`

Reply:
Actually no.
This code path is very broken.
We need to better define Clone() method.

Reply:
good catch!